test/test_testing.py

```diff
@@ -798,42 +798,80 @@ def test_mismatching_values(self, device):
 
     @onlyCPU
     def test_mismatching_values_msg_abs_mismatches(self, device):
-        a = torch.empty((3, 4), dtype=torch.float32, device=device).fill_(5)
+        a = torch.empty((3, 3), dtype=torch.float32, device=device).fill_(5)
```

Comment:
Prefer zeros to empty, since empty could be anything (including the correct value)

Reply:
1. Shouldn't this make no difference given that we are invoking `fill_()` after the instantiation?
2. Why do you want to change the size?

Reply:
Good point. I missed the fill_, sorry.

In that case uses torch.full().

I'm not sure I understand what you mean about changing the size?

Reply:
> I'm not sure I understand what you mean about changing the size?

My bad, I misread the diff.